MemoryLeak.Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM node:22-alpine3.20
+
+RUN apk add --no-cache \
+    udev \
+    ttf-freefont \
+    chromium \
+    python3\
+    make \
+    g++ \
+    && npm install -g pnpm@10.6.5
+
+WORKDIR /app
+COPY .puppeteerrc.cjs /app/.puppeteerrc.cjs
+COPY . .
+
+ENV PUPPETEER_CONFIG_FILE="/app/.puppeteerrc.cjs" \
+    PUPPETEER_SKIP_CHROMIUM_DOWNLOAD=true \
+    PUPPETEER_EXECUTABLE_PATH=/usr/bin/chromium
+
+RUN make install
+
+# Cleanup old profile locks and ensure proper directory permissions
```

Comment:
Please remove all unnecessary comments from the code
P.S. check all files that you changed